### PR TITLE
Feature/iat 3542

### PR DESCRIPTION
**Pull request by bdrainer**
The primary change is making the method "public" in the EbsrTransformer.  The TIMS scoring service uses the transformers to generate the QRX as a string. All, except Ebsr, was already public.

The other change is adding a configuration for the typical beans needed to generate SAAIF files and item.json.  Most services configure those beans in their respective app level configuration.  It seems like a good idea to have this.

### CODE REVIEW

**Comment by bdrainer** [comment]
I'll merge now but feedback is welcome and I will apply the suggestions as soon as I can and get a new PR up with them.

File: src/main/java/org/opentestsystem/ap/common/config/SaaifConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public JsonModelAssembler jsonModelAssembler() {
+        log.info("Creating bean jsonModelAssembler");
+        return new JsonModelAssembler();
+    }
+
+    // ------------------------------------------------------------------------
```
Comment (millerts) [inline src/main/java/org/opentestsystem/ap/common/config/SaaifConfiguration.java]: What does this line represent?

File: src/main/java/org/opentestsystem/ap/common/config/SaaifConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        this.restTemplateBuilder = restTemplateBuilder;
+        this.freemarker = freemarker;
+    }
+
+    // ------------------------------------------------------------------------
+
+    @Bean
+    @ConditionalOnMissingBean
+    public ImageGenerator imageGenerator() {
+        log.info("Creating bean imageGenerator");
```
Comment (millerts) [inline src/main/java/org/opentestsystem/ap/common/config/SaaifConfiguration.java]: Can we change all the logging in this class to debug?

File: src/main/java/org/opentestsystem/ap/common/config/SaaifConfiguration.java (hunk 1 above)
Comment (bdrainer) [reply src/main/java/org/opentestsystem/ap/common/config/SaaifConfiguration.java]: I deleted these.  We spoken about these before.

File: src/main/java/org/opentestsystem/ap/common/config/SaaifConfiguration.java (hunk 2 above)
Comment (bdrainer) [reply src/main/java/org/opentestsystem/ap/common/config/SaaifConfiguration.java]: I like info here.  This only happens on startup and I like seeing the beans as they are created.  Some of the other config classes are doing this like DatabaseConfiguration.

**Pull request merged**